### ISSUE TITLE
[Bug]: Undefined array key "dontCheckEnabled"

### ISSUE THREAD

**Issue by LosHawlos** [pimcore/pimcore#12002]
### Expected behavior

Areabricks can be enabled/disabled without showing errors in the Website

### Actual behavior

When an Areabrick is used inside a pimcore_areablock, and the Areabrick the later gets disabled using Tools / Bundles&Bricks the following error is shown on the website:

```
Warning: Undefined array key "dontCheckEnabled"
File: /var/www/html/vendor/pimcore/pimcore/models/Document/Editable/Areablock.php Line: 186
#0 /var/www/html/vendor/pimcore/pimcore/models/Document/Editable/Areablock.php(106): Pimcore\Model\Document\Editable\Areablock->loop()
#1 /var/www/html/vendor/pimcore/pimcore/models/Document/Editable.php(473): Pimcore\Model\Document\Editable\Areablock->frontend()
#2 /var/www/html/vendor/pimcore/pimcore/models/Document/Editable.php(486): Pimcore\Model\Document\Editable->render()
#3 /var/www/html/var/cache/dev/twig/34/3424b5fd441e5f788fa65dc29f7ec056.php(46): Pimcore\Model\Document\Editable->__toString()
#4 /var/www/html/vendor/twig/twig/src/Template.php(394): __TwigTemplate_562cf6df90a56f397edf8c76b2e764fa->doDisplay(Array, Array)
#5 /var/www/html/vendor/twig/twig/src/Template.php(367): Twig\Template->displayWithErrorHandling(Array, Array)
#6 /var/www/html/vendor/twig/twig/src/Template.php(379): Twig\Template->display(Array)
#7 /var/www/html/vendor/twig/twig/src/TemplateWrapper.php(40): Twig\Template->render(Array, Array)
#8 ... ... ...

```

### Steps to reproduce

Tested on Pimcore Version 10.4.0

- Add an Areabrick into an Areablock and save the Document.
- The Website is shown properly
- Disable the Areabrick in Tools/Bundles&Bricks
- Reload the page and the error is show.

**Comment by LosHawlos** [comment]
I forgot to mention: The error is only shown when APP_DEBUG is set to true.
With APP_DEBUG=false no error is visible.

**Comment by brusch** [comment]
Fixed by #12003
